packages/cfg-core/lib/util/DeepCopy.ts

```diff
@@ -0,0 +1,33 @@
+export function deepCopy<T>(instance: T): T {
```

Comment:
Did you write this piece of code, or did it come from somewhere?

Reply:
lodash has a package lodash.deepCopy.
We should import this package instead of creating this function ourselves.